### ISSUE TITLE
Adding a "closes" or "fixes" to a rebased commit message does not attach PR to issue in Waffle UI

### ISSUE THREAD

**Issue by caseypt** [waffleio/waffle.io#1808]

**Comment by caseypt** [comment]
If you forget to add a "closes" or "fixes" in a commit message, then you rebase or amend and add the "closes" or "fixes", you have to close and create a new PR with the updated commits to get Waffle to attach the PR to the referenced issue.

Ideally you should also be able to make a comment on the PR with "closes" or "fixes" to have Waffle attach the issue and PR. Github respects this as another way to autoclose an issue.

**Comment by ashumz** [comment]
@caseypt You can add the closes or fixes keywords in front of an issue number to  a pull request's title or description at any time to visually connect that pull request to the issue and have the issue closed once the PR is merged into the default branch for your repository. 

We don't look at commits yet, but if you edit the PR title or description in Waffle after the fact, you can still achieve this same functionality by editing the PR in Waffle. Note that the issue referenced in the PR will become visually connected/nested beneath it (hopefully that's a good thing for your workflow :)). 

Let us know if you have any questions about this! We have more details around how this works on our wiki here: https://github.com/waffleio/waffle.io/wiki/FAQs#prs-and-issues

**Comment by ashumz** [comment]
One additional thing to note -- if you change the PR description or title via Waffle, you may have to refresh the board before the connection appears.

**Comment by caseypt** [comment]
Thanks!
